evaluator/src/main/kotlin/PackageRule.kt

```diff
@@ -70,6 +70,27 @@ open class PackageRule(
         licenseRules.forEach { it.evaluate() }
     }
 
+    /**
+     * A [RuleMatcher] that checks whether a severity of any vulnerability of the [package][pkg] using the specified
+     * [scoringSystem] is higher than [maxValue]. Checked using the [severityComparator]. Returns false if the severity
+     * is equal to [maxValue].
```

Comment:
How about this instead:

A [RuleMatcher] that checks whether any vulnerability for the [package][pkg] exceeds [maxValue] according to the [scoringSystem] and the belonging [severityComparator].